src/components/LabeledDropdown.module.css

```diff
@@ -5,11 +5,39 @@
   gap: 6px;
 }
 
-.labeledDropdown button {
+/* Make button and dropdown have the same width */
+#dropdownButton,
+.dropdownContent {
```

Comment:
Why is `dropdownButton` an id rather than a class? Ids should be unique items which appear only once in the document.

Reply:
`!important` is kind of a last-resort escape hatch, which is forgivable in certain cases (I've used it a couple times over in website-3d-cell-viewer). Ids let the browser assume that an element is unique and treat it differently, so it's probably worse practice not to preserve that. If turning these back into classes means you instead have to stick `!important` onto every style rule, that's not nice and we should try to find a secret third option, but I think I'd prefer to merge this with `!important` all over the place rather than assigning the same id to a list of elements.